strapi/strapi
# [Custom fields] Add custom field input component in CM edit view

### What does it do?

- Lazy loads a custom field's input component in the content manager's edit view
- Updates the sample color picker component as a POC

### Why is it needed?

To let custom fields use their own component to manage the state of a custom field attribute

### How to test it?

- Open `/admin/content-manager/collectionType/api::kitchensink.kitchensink/create`, see the `custom_field` attribute
- Click on the label and it should open the input
- Modify and save the color, the value should persist after refreshing the page
